packages/jest-util/src/Console.js

```diff
@@ -127,7 +127,7 @@ export default class CustomConsole extends Console {
 
     if (startTime) {
       const endTime = new Date();
```

Comment:
This should be using `process.hrtime`, not `Date`. Not related to this PR, though...